main.py

```diff
@@ -26,59 +31,79 @@ def validate_number(num):
         return False
 
 
-def validate_operator(operator):
+def validate_operator(operator: any) -> bool:
```

Comment:
similar thing is here. you're getting a string in argument so: (operator: str) is enough
if we decided to add all this typing let's make it as much accurate as possible